span-normalizer/span-normalizer/src/main/resources/configs/common/application.conf

```diff
@@ -6,6 +6,7 @@ main.class = org.hypertrace.core.spannormalizer.SpanNormalizer
 span.type = jaeger
 input.topic = "jaeger-spans"
 output.topic = "raw-spans-from-jaeger-spans"
+bypass.output.topic = "structured-traces-from-raw-spans"
```

Comment:
maybe we should call it `structured-traces-topic` (or the name thats used in grouper) to indicate that this not a different topic

Reply:
Had kept as per the key configuration - `bypass.key`. Do you have suggestions for that too?